.github/workflows/build-bottles.yml

```diff
@@ -0,0 +1,61 @@
+name: Build bottles.
+
+on:
+  pull_request:
+    paths:
+      - 'Formula/*'
+
+jobs:
+  build-bottles:
+    strategy:
+      matrix:
+        version: [10.15, 10.14, 10.13]
+    runs-on: ${{ matrix.version }}
+    env:
+      HOMEBREW_GIT_NAME: BrewTestBot
+      HOMEBREW_GIT_EMAIL: 'homebrew-test-bot@lists.sfconservancy.org'
+      PATH: '/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin'
+      HOMEBREW_DEVELOPER: 1
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - name: Setup tap
```

Comment:
Could this use actions/checkout@master?

Reply:
This copies the Jenkins configuration. I believe test bot looks at the current brew repository, not its current directory.

Reply:
I'm not sure I understand this, can you elaborate a bit?